PerfettoCds/Protobuf/Descriptor.cs

```diff
@@ -0,0 +1,2797 @@
+// <auto-generated>
+//     Generated by the protocol buffer compiler.  DO NOT EDIT!
+//     source: descriptor.proto
+// </auto-generated>
+#pragma warning disable 1591, 0612, 3021
+#region Designer generated code
+
+using pb = global::Google.Protobuf;
+using pbc = global::Google.Protobuf.Collections;
+using pbr = global::Google.Protobuf.Reflection;
+using scg = global::System.Collections.Generic;
+namespace Perfetto.Protos {
```

Comment:
Maybe put a note in the same dir describing how this was generated or could be re-generated

Reply:
Added a readme for these